client/layout/masterbar/woo.scss

```diff
@@ -310,6 +310,15 @@
 		padding-left: 0;
 		padding-right: 0;
 		background-color: initial;
+		&.gdpr-banner {
```

Comment:
Shouldn't changes like these get implemented in gdpr banner directly instead, so to harden it and avoid having related CSS in many different places? CSS here makes updating the banner harder and prone for future breakage.

Alternatively Woo page's CSS could be scoped more specifically to avoid leaking into banners. I.e. not `.card` but `.card.woo-something`.

Reply:
> Shouldn't changes like these get implemented in gdpr banner directly instead, so to harden it and avoid having related CSS in many different places? CSS here makes updating the banner harder and prone for future breakage.

Good point! 👍 

> Alternatively Woo page's CSS could be scoped more specifically to avoid leaking into banners. I.e. not .card but .card.woo-something.

If for some reason we cannot use .card.woo-something, I think we could use `:not()` to exclude `gpdr-banner`. 

cc @moon0326

Reply:
> `:not()`

Also a good idea 👍

Reply:
Another way to scope `.woo .card` tighter is to target only `.layout__content .card` but it's still a bit too global to my taste.

Reply:
Thanks for the feedback! 

> Alternatively Woo page's CSS could be scoped more specifically to avoid leaking into banners. I.e. not .card but .card.woo-something.

I think it's difficult to add more `woo` context to the html structure without complicating the component logic since the `wpcc` page is heavily shared.

> If for some reason we cannot use .card.woo-something, I think we could use :not() to exclude gpdr-banner

I'm not sure if I understand here, do you mean to use `:not(.gdpr-banner)` for the base `.card` class?

> Shouldn't changes like these get implemented in gdpr banner directly instead, so to harden it and avoid having related CSS in many different places? CSS here makes updating the banner harder and prone for future breakage.

To understand this better, you meant moving the CSS to https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/gdpr-banner/style.scss with adding `.woo` class correct?